src/Portal.js

```diff
@@ -102,11 +102,7 @@ let Portal = React.createClass({
     }
 
     if (this._overlayInstance) {
-      if (this._overlayInstance.getWrappedDOMNode) {
-        return this._overlayInstance.getWrappedDOMNode();
```

Comment:
Just compat with the old RCW... this method was only defined on RCW and doing a quick GH code search, the only instances of it are people vendoring this code